docs/kratos/passwordless/01_webauthn.mdx

```diff
@@ -68,8 +68,10 @@ import TabItem from '@theme/TabItem';
           id: example.org
           # This MUST be the exact URL of the page which will prompt for WebAuthn!
           # Only the scheme (https / http), host (auth.example.org), and port (4455) are relevant. The
-          # path is irrelevant
-          origin: http://auth.example.org:4455
+          # path is irrelevant.
+          origins:
```

Comment:
is this a breaking change?

Reply:
No, the old origin config value is still valid, but if "origins" is set it takes precedences.